test/vm.install

```diff
@@ -4,9 +4,10 @@
 set -eu
 
 # don't force https:// (self-signed cert)
+mkdir -p /etc/cockpit
 printf "[WebService]\\nAllowUnencrypted=true\\n" > /etc/cockpit/cockpit.conf
 
 if type firewall-cmd >/dev/null 2>&1; then
-    firewall-cmd --add-service=cockpit --permanent
+    firewall-cmd --add-service=cockpit --permanent || true
```

Comment:
Why would this fail? Maybe it needs an additional `systemctl start firewalld`, or some `if systemctl is-active` instead of the `type firewalld`? I'm generally allergic to `|| true` in tests, although this is a harmless variant.

Reply:
Should we really start the firewall if we don't need to?

Reply:
No, hence the proposal with checking `is-active`

Reply:
Good call.  I'll take a look.